src/components/groups/GroupContainer.js

```diff
@@ -35,13 +36,21 @@ function GroupContainer() {
 export default GroupContainer;
 
 const Container = styled.div`
+@media ${device.laptop} {
```

Comment:
width should be something like 95% for the desktop view, too much visible margin at 80% with 1024 pixels